README.md

```diff
@@ -49,6 +49,8 @@ $ sudo GOPATH=$GOPATH CNI_PATH=$GOPATH/bin ./kube-spawn setup --image=coreos --n
 $ sudo GOPATH=$GOPATH CNI_PATH=$GOPATH/bin ./kube-spawn init
 ```
 
+Now that you're up and running, this is how you can start using it.
+
```

Comment:
@robertgzr I just changed it simply to `"Now that you're up and running, you can start using it."`